src/components/global/SiteLayout/index.tsx

```diff
@@ -45,6 +45,24 @@ const SiteLayout = ({ pageMeta, children }) => {
     defaultContext.activate(network)
   }, [])
 
+  if (
+    pageMeta.url ===
```

Comment:
Any reason this wouldn't be global?

Reply:
These 2 pages didn’t need header and footer.